pandas/tests/groupby/test_nth.py

```diff
@@ -517,11 +517,11 @@ def test_nth_multi_index_as_expected():
 @pytest.mark.parametrize(
     "op, n, expected_rows",
     [
-        ("head", -1, []),
+        ("head", -1, [0]),
```

Comment:
these are essentially bug fixes?

Reply:
Essentially. Although the docstring says "Does not work for negative values of n." and shows the empty frame returned in the example. So you could argue that the old behaviour was by design and the new one is therefore an enhancement...